tests/unit/test_consolidator.py

```
@@ -0,0 +1,708 @@
+        assert result.checked_count == 1  # archived は含まない
+
+
+# ---------------------------------------------------------------------------
+# 3. 優先順位テスト（自己修復 >= 0.90 を通常統合 0.85–0.89 より優先）
+# ---------------------------------------------------------------------------
+
+class TestPriorityProcessing:
+    """優先順位テスト: 自己修復候補を通常統合候補より優先して処理することを検証。"""
+
+    async def test_self_healing_prioritized_over_regular_consolidation(self):
+        """0.90以上の自己修復候補が 0.85–0.89 の通常候補より先に処理されること。
```

Ruff (0.15.7)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)